packages/client/components/ScopingSearchBar.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from '@emotion/styled'
+import React, {ReactNode} from 'react'
+import {PALETTE} from '../styles/paletteV3'
+
+const SearchBar = styled('div')({
+  padding: 16
+})
+
+const SearchBarWrapper = styled('div')({
+  alignItems: 'center',
+  border: `1px solid ${PALETTE.SLATE_400}`,
+  borderRadius: '40px',
+  display: 'flex',
+  height: 44,
+  padding: '0 16px',
+  width: '100%'
+})
+
+const CurrentFiltersWrapper = styled('div')({
+  width: '100%',
+  display: 'flex',
+  paddingLeft: '72px',
+  paddingTop: '8px'
+})
+
+const Description = styled('div')({
+  color: PALETTE.SLATE_600,
+  fontSize: 16,
+  fontWeight: 500,
+  whiteSpace: 'nowrap'
+})
+
+const Items = styled('div')({
+  color: PALETTE.SLATE_600,
+  fontSize: 16,
+  fontWeight: 600,
+  fontStyle: 'italic',
+  padding: '0px 24px 0px 4px',
+  width: '100%',
+  whiteSpace: 'nowrap',
+  overflow: 'hidden',
+  textOverflow: 'ellipsis'
+})
+
+interface Props {
+  children: ReactNode
+  currentFilters?: string
+}
+
+const ScopingSearchBar = (props: Props) => {
```

Comment:
+2 lovely stuff making this reusable! 

+1 It looks like you've implemented this in Jira, but GitHub & GitLab are using the old version. If you want to prevent PR bloat and don't want to change them in this PR, perhaps we should create an issue to change them? Otherwise, someone might edit the `ScopingSearchBar`, see that it's updated the UI in Jira & Jira Server, and assume it's done the same for the other integrations.

Reply:
@nickoferrall created an issue to keep track of it #6435